examples/server/main.go

```diff
@@ -65,3 +93,51 @@ func newAWSPubSubServer() (pubsubgrpc.PubSubServer, error) {
 	}
 	return pubsubgrpc.NewPubSubServer(pubFactory, subFactory), nil
 }
+
+// ServeInternal builds and runs the server that listens on InternalAddress
+func ServeInternal(logger *logrus.Logger) error {
+	healthChecker := health.NewChecksHandler(
+		viper.GetString("internal.health"),
+		viper.GetString("internal.readiness"),
+	)
+	healthChecker.AddLiveness("ping", health.HTTPGetCheck(
+		fmt.Sprint("http://", viper.GetString("internal.address"), ":", viper.GetString("internal.port"), "/ping"), time.Minute),
```

Comment:
I see similar thing in contacts app. Not sure that it is a proper way to check liveness. Can we think of some better way?